clusterloader2/pkg/measurement/common/slos/network_programming.go

```diff
@@ -35,7 +36,7 @@ const (
 	// %v should be replaced with query window size (duration of the test).
 	// This measurement assumes, that there is no data points for the rest of the cluster-day.
 	// Definition: https://github.com/kubernetes/community/blob/master/sig-scalability/slos/network_programming_latency.md
-	query = "quantile_over_time(0.99, kubeproxy:kubeproxy_network_programming_duration:histogram_quantile{quantile='0.99'}[%v])"
+	query = "quantile_over_time(0.99, kubeproxy:kubeproxy_network_programming_duration:histogram_quantile{}[%v])"
```

Comment:
Just to confirm, what we'll have in perf-dash will be 
`99th percentile of all measurements of Xth percentiles last 5min, where X=50,90,99`

Right?

Reply:
Yes, that's right. 

Shall I update the comment to be more clear?

Reply:
Would be great, but can be done in a follow up PR. This looks good, don't want to block it anymore